doc/sphinx/proofs/automatic-tactics/index.rst

```diff
@@ -13,6 +13,8 @@ complex goals in new domains.
 .. toctree::
    :maxdepth: 1
 
+   auto
+   decision
```

Comment:
>reverse the order to "Programmable tactics and ..."

Yes.

>6 sections on decision procedures and only one on programmable tactics

No, "Generalized rewriting" is also about programmable tactics. Should it move up in the list of sections?

>Perhaps out of scope for this PR, but the section names here could be a bit more consistent and allude to their use case, e.g. leave out "Decision procedures", just say "For logic and equality"; "Omega: for quantifier-free ..."; "Micromega: for solving"; "for rings and fields"

Great suggestion!

> Not sure we need to introduce the term "decision procedure" which AFAIK we don't really use. The point is that it's an _algorithm_ not just a tactic. Could just say something like: Provided the goal is in the appropriate form for the tactic, it will prove the goal if that is possible. If the tactic doesn't prove the goal, then the goal is unprovable.

Yes, but what would you put in the *title* of this chapter? To me the idea was that it was about "automatic tactics", which are of two kinds: built-in automation to solve specific types of goals (a.k.a., decision procedures), and user-programmable proof search tactics.

Reply:
> "Generalized rewriting" is also about programmable tactics. Should it move up in the list of sections?

I dunno.  I don't know too much about any of these.  I hope we can get more introductory material soon describing their purpose/when to use them/why users would be interested in them.

> Yes, but what would you put in the *title* of this chapter?

For the ones that are indeed algorithms (failure means not provable), they could be called "solvers".  Does "decision procedure" mean that?

Reply:
So I will go indeed for your "solver" suggestion instead of the less transparent "decision procedure". And I agree with adding some consistency to the section titles, but repeating "solver" everywhere is important so that the page titles are standalone.